tests/benchmarks/micro_benchmarks/test_nvbandwidth.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Tests for nvbandwidth benchmark."""
+
+import unittest
+
+from tests.helper.testcase import BenchmarkTestCase
+from superbench.benchmarks import BenchmarkRegistry, ReturnCode, Platform
+
+
+class TestNvBandwidthBenchmark(BenchmarkTestCase, unittest.TestCase):
+    """Test class for NV Bandwidth benchmark."""
+    @classmethod
+    def setUpClass(cls):
+        """Hook method for setting up class fixture before running tests in the class."""
+        super().setUpClass()
+        cls.createMockEnvs(cls)
+        cls.createMockFiles(cls, ['bin/nvbandwidth'])
+
+    def test_nvbandwidth_result_parsing_real_output(self):
+        """Test NV Bandwidth benchmark result parsing."""
+        benchmark_name = 'nvbandwidth'
+        (benchmark_class,
+         predefine_params) = BenchmarkRegistry._BenchmarkRegistry__select_benchmark(benchmark_name, Platform.CUDA)
+        assert (benchmark_class)
+
+        benchmark = benchmark_class(benchmark_name, parameters='')
+
+        # Preprocess and validate command
+        assert benchmark._preprocess()
+
+        # Provided raw output
+        raw_output = """
+        nvbandwidth Version: v0.6
```

Comment:
can you move the log to `tests/data` and use `@decorator.load_data` to load here?